app/assets/javascripts/components/PipelineSampleReport.jsx

```diff
@@ -180,23 +218,29 @@ class PipelineSampleReport extends React.Component {
     $(`.table-arrow`).toggleClass('hidden');
   }
 
+  //Download report in csv
+  downloadReport(id) {
+    location.href = `/reports/${id}/csv`
+  }
+
   render() {
     const parts = this.props.report_page_params.sort_by.split("_")
     const sort_column = parts[1] + "_" + parts[2];
     var t0 = Date.now();
-    filter_stats = this.rows_passing_filters + ' rows passing filters, out of ' + this.rows_total + ' total rows.';
+    filter_stats = this.rows_passing_filters + ' Results matching filter criteria';
```

Comment:
It's really important to know how many rows did *not* match the criteria.   Is that information being removed?

Reply:
thanks, the new design had only the matching filters, but would revert back to show all as well.